File: bronze/2577/c++/main.cpp
#include <iostream>

int main()
{
    int a, b, c;
    std::cin >> a >> b >> c;
    long long int result = a * b * c;
    int answer[10] = {0};
    while (result > 0) {
        answer[result % 10]++;
        result /= 10;
    }

    for (int i : answer) {
        std::cout << i << '\n';
    }

    return 0;
}

File: bronze/2577/python/main.py
 
if (__name__ == "__main__"):
    a = int(input())
    b = int(input())
    c = int(input())
    result = a * b * c
    answer = [0] *10
    while (result > 0):
        answer[result % 10] += 1
        result = int(result / 10)
    
    print('\n'.join(map(str, answer)))